k8s/common/mailrelay.yaml

```diff
@@ -0,0 +1,5 @@
+---
+apiVersion: v1
+kind: Namespace
```

Comment:
can you create this as part of the kustomize base, `common` shouldn't be used for anything new

Reply:
Yes, had not seen that. Thanks